### PR TITLE
SmashingMagazine - First of two articles

### PR DESCRIPTION
**Pull request by MarcoHengstenberg**
Now, that all the data is right, you may pull whenever you have the time. =)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -52,6 +52,21 @@
...
+</h3>
+<p class="author">
+<i>by</i> <a href="https://www.twitter.com/retinart">Alexander Charchar</a>
+</p>
+<p>We’ve all experienced that burnout moment. It’s that moment when we’ve got nothing left to give but keep trying anyway, when we’re left without much more than a shell to live in and motions to go through.</p>
+
+<p>We’re fried and broken and wish desperately for our work to make sense, for our energy to come back, for things to be fun and as they were. In such moments all we want is for our work to feel like our work and not like torture.</p>
+
+<p>“Are you all right? You’re a little gray,” our friends and loved ones say with concern. “You should get some rest.”</p>
+</p>
```
Comment (mikebell) [inline index.html]: Please can you remove the extra tag.

Comment (MarcoHengstenberg) [inline index.html]: I'm sorry but it seems I can't see the obvious.
What tag needs to be removed? The whole part from line 55 to 68 or just the surrounding `<div>`?

Thought I'd followed the guidelines. :frowning:

Comment (mikebell) [inline index.html]: Line 68, according to PHPStorm "Closing tag matches nothing"

I'll take a look at the copy and paste snippet to see if there is anything wrong there.

Comment (MarcoHengstenberg) [inline index.html]: Aaaah I found the bugger. Sorry, my fault. Fixing on the way.

**Comment by mikebell** [comment]
Thanks!

**Comment by MarcoHengstenberg** [comment]
I have to thank you for taking care. =)
Have a nice day Mike!

**Pull request merged**